### PR TITLE
added support for nio

### PR DESCRIPTION
**Pull request by prashantabmc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,23 +17,22 @@ Collects metrics from a Apache Tomcat instances
 
 ### Plugin Setup
 
-In order for the plugin to collect statistics from Tomcat you need to configure  JMX endpoint for Tomcat installed.To set the CATALINA_OPTS environment variable (assuming JMX endpoint as  8999 of your localhost)
+In order for the plugin to collect statistics from Tomcat you need to configure  JMX endpoint for Tomcat installed.To set the CATALINA_OPTS environment variable(assuming JMX endpoint as  8999 of your localhost)
```
Comment (gokumar) [inline README.md]: Please provide space after '.' (full stop)

File: README.md [hunk 2]
```diff
@@ -17,23 +17,22 @@ Collects metrics from a Apache Tomcat instances
...
 
-- On Windows (if Tomcat is running as service):
-        - Enable Apache Service Manager (commons daemon service manager) for the installed service using the  command:  tomcat7w.exe
-
-	        This should start Apache Service Monitor program on your system tray. Click on its icon. select  on the 'Java' tab and append the following on the 'Java Options' text box, one option per line:
-
-		    -Dcom.sun.management.jmxremote.port=8999
-		    -Dcom.sun.management.jmxremote.authenticate=false
-		    -Dcom.sun.management.jmxremote.ssl=false
+- On Windows ( if Tomcat is running as service ):
```
Comment (gokumar) [inline README.md]: no spaces after '(' and before ')'

File: README.md [hunk 3]
```diff
@@ -49,9 +48,9 @@ In order for the plugin to collect statistics from Tomcat you need to configure
 ### Metrics Collected
 
 | Metric Name | Description |
-|:------------|:------------|
-|TOMCAT_JVM_FREE_MEMORY | Free memory of the JVM in MBytes.|
-|TOMCAT_JVM_TOTAL_MEMORY | Total memory of the JVM in Mbytes.|
```
Comment (gokumar) [inline README.md:54]: Why are these removed?

Comment (prashantabmc) [inline README.md:54]: Not available through JMX.
